native/src/components/Feedback.tsx

```diff
@@ -133,26 +134,39 @@ const Feedback = ({
               <FeedbackButtons isPositiveFeedback={isPositiveFeedback} setIsPositiveFeedback={setIsPositiveFeedback} />
             </>
           )}
+
           <HeadlineContainer>
             <Description>{t('commentHeadline')}</Description>
             <Text>({t('optionalInfo')})</Text>
           </HeadlineContainer>
           <DescriptionContainer>
             <Text>{t('commentDescription', { appName: buildConfig().appName })}</Text>
           </DescriptionContainer>
-          <CommentInput onChangeText={onCommentChanged} value={comment} multiline numberOfLines={3} />
+          <CommentInput
+            onChangeText={onCommentChanged}
+            value={comment}
+            multiline
+            numberOfLines={3}
+            blurOnSubmit
```

Comment:
Why did you decide to only blur this one on submit?

Reply:
because i want to leave the textfield? what would you suggest?

Reply:
I'm fine with that, I just figure that it should then blur the other inputs as well.

Reply:
Basically because its default true for single line inputs

_If true, the text field will blur when submitted. The default value is true for single-line fields and false for multiline fields. Note that for multiline fields, setting blurOnSubmit to true means that pressing return will blur the field and trigger the onSubmitEditing event instead of inserting a newline into the field._

Reply:
Why is blurring the text input connected to leaving the input field?

Reply:
because you loose the focus of the field

Reply:
Thanks for explaining, makes sense!